### PR TITLE
breaks out profiles into reasonably-sane modules that can be worked on seperately

### PR DESCRIPTION
**Pull request by owocki**
breaks out profiles into reasonably-sane modules that can be worked on seperately

@danlipert this is one of the first big changes to go into `profile/v2` so PRing it seperately so we can reason about it here.

### CODE REVIEW

File: app/dashboard/templates/profiles/scorecard_funder.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      </div>
+      <div class="card-body font-subheader">
+        <ul>
+          <li><b>{{ funded_bounties_count }}</b> {% trans "bounties funded" %}</li>
+          <li><b>{{ sum_eth_funded|floatformat:2 }} ETH</b> {% trans "funded" %}</li>
+          <li class="works_with_list">
+            {% if works_with_funded|length %}
+              <span class="font-body"><i class="fab fa-ethereum"></i> {% trans "Funds" %}:</span>
+              {% for profile, num_times in works_with_funded.items %}
+                {% if forloop.counter < 6 %}
```
Comment (danlipert) [inline app/dashboard/templates/profiles/scorecard_funder.html]: can use a template tag here `|slice:":5"` instead of the if check

Comment (owocki) [reply app/dashboard/templates/profiles/scorecard_funder.html]: hmm... this is code that i didnt write (i just moved location).. will take a look tho

File: app/dashboard/templates/profiles/header_details.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                  <i class="fa fa-briefcase mr-2" aria-hidden="true"></i>{{ profile.job_status_verbose }}
+                  {% if profile.resume %}<a href="{{ MEDIA_URL }}{{profile.resume}}" download>View Resume</a>{% endif %}
+                </div>
+              {% endif %}
+              {% if profile.data.bio %}
+                <div class="profile-header__bio my-2">
+                  {{ profile.data.bio }}
+                </div>
+              {% endif %}
+            </div>
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/header_details.html]: ^ @owocki would you be open to setting up a linter into the editor ? 😅

Comment (owocki) [reply app/dashboard/templates/profiles/header_details.html]: i have anaconda setup. i think thats python only.  i guess `make fix` doesnt lint html?

**Pull request merged**